src/main/java/com/limechain/sync/warpsync/SyncedState.java

```diff
@@ -0,0 +1,64 @@
+package com.limechain.sync.warpsync;
+
+import com.limechain.chain.lightsyncstate.Authority;
+import com.limechain.constants.GenesisBlockHash;
+import com.limechain.network.Network;
+import com.limechain.network.protocol.blockannounce.scale.BlockAnnounceHandshake;
+import com.limechain.rpc.http.server.AppBean;
+import io.emeraldpay.polkaj.types.Hash256;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigInteger;
+
+public class SyncedState {
+    private static final SyncedState INSTANCE = new SyncedState();
+    @Getter
+    @Setter
+    public Hash256 lastFinalizedBlockHash;
+    @Getter
+    @Setter
+    public Hash256 stateRoot;
+    @Getter
+    @Setter
+    public BigInteger lastFinalizedBlockNumber = BigInteger.ZERO;
+    @Getter
+    @Setter
+    public Authority[] authoritySet;
+    @Getter
+    @Setter
+    public BigInteger setId;
+    @Getter
+    @Setter
+    public byte[] runtime;
+    @Getter
+    @Setter
+    public byte[] heapPages;
+
+    public static SyncedState getINSTANCE() {
+        return INSTANCE;
+    }
```

Comment:
I think the function should be camel case regardless of the field being fully capitalized.

Reply:
Yes, not sure how it turned out to be this way